scripts/monitoring/cron-send-create-app.py

```diff
@@ -65,6 +72,8 @@ def parse_args():
     parser.add_argument('--basename', default="test", help='base name, added to via openshift')
     parser.add_argument('--namespace', default="ops-health-monitoring",
                         help='namespace (be careful of using existing namespaces)')
+    parser.add_argument('--loopcount', default="36", 
+                        help="how many 5 second loops before giving up on app creation")
```

Comment:
optional: 5 9the number) may not be true in the future, and having it hardcoded can become an error in the future .. I'm thinking this would be more clearly described if we had all the params specified